core/src/util.jl

```diff
@@ -707,3 +707,25 @@ function Base.get(
         nothing
     end
 end
+
+"""
+Get the time interval between (flow) saves
+"""
+function get_Δt(integrator, graph)::Float64
```

Comment:
Probably better to make this accept only `integrator` and take the graph out of the parameters.